packages/near-membrane-base/src/red.ts

```diff
@@ -185,7 +183,7 @@ export const serializedRedEnvSourceText = /* prettier-ignore */ (function redEnv
     }
 
     function getBluePartialDescriptor(redPartialDesc: PropertyDescriptor): PropertyDescriptor {
-        const bluePartialDesc = ObjectAssign(ObjectCreate(null), redPartialDesc);
+        const bluePartialDesc = { __proto__: null, ...redPartialDesc };
```

Comment:
Since there is a PropertyDescriptorMap is there a PropertyDerscriptor we should be using?